packages/venia-concept/src/RootComponents/CMS/CMS.js

```diff
@@ -1,8 +1,62 @@
 import React, { Component } from 'react';
-import CategoryList from 'src/components/CategoryList';
+import { Form } from 'informed';
```

Comment:
The changes to this file will not be merged; they're for demonstration purposes only. When this PR is approved, I'll revert this file before merging.